napari_btrack/track.py

```diff
@@ -0,0 +1,47 @@
+import btrack
+import napari
+
+from btrack.utils import segmentation_to_objects
+from magicgui import magicgui
+from magicgui.widgets import FunctionGui
+from pathlib import Path
+from typing import Optional
+
+def run_tracker(objects, config_file_path):
+    with btrack.BayesianTracker() as tracker:
+        # configure the tracker using a config file
+        tracker.configure_from_file(config_file_path)
+        tracker.max_search_radius = 50
+
+        # append the objects to be tracked
+        tracker.append(objects)
+
+        # set the volume
+        tracker.volume=((0, 1600), (0, 1200), (-1e5, 64.))
+
+        # track them (in interactive mode)
+        tracker.track_interactive(step_size=100)
+
+        # generate hypotheses and run the global optimizer
+        tracker.optimize()
+
+        # get the tracks in a format for napari visualization
+        data, properties, graph = tracker.to_napari(ndim=2)
+        return data, properties, graph
+
+def track() -> FunctionGui:
+    @magicgui(
+        call_button=True,
+        persist=True,
+        config_file_path=dict(value=Path.home()),
+        reset_button=dict(widget_type="PushButton", text="Reset defaults"),
+    )
+    def widget(
+        viewer: napari.Viewer,
+        segmentation: napari.layers.Image,
+        config_file_path: Optional[Path],
+        reset_button):
+        segmented_objects = segmentation_to_objects(segmentation.data[:100, ...])   
```

Comment:
```suggestion
        segmented_objects = segmentation_to_objects(segmentation.data)   
```
Presumably we don't want to be chopping up the data here?

Reply:
for now yes, but we will set that with a config parameter.